mlflow/server/handlers.py

```diff
@@ -799,6 +818,52 @@ def _delete_model_version_tag():
     return _wrap_response(DeleteModelVersionTag.Response())
 
 
+@catch_mlflow_exception
+def _download_artifact(artifact_path):
+    basename = os.path.basename(artifact_path)
```

Comment:
artifact paths for the `mlflow-artifacts` REST API are always given in POSIX path format, so I think we need to use `posixpath.basename` for parsing. When constructing local filesystem paths (e.g. 2 lines below), `os.path` is still appropriate.

Reply:
True, let me fix this.